deploy/file_list

```diff
@@ -1,8 +1,10 @@
-Assets/Plugins/Android/SoomlaAndroidCore.jar
```

Comment:
@borisspektor, I fixed it for Core module, but not sure that we can do it with others, because there is symlinks which shouldn't be packed into unitypackage.